src/styles/utils/_mixins.scss

```
@@ -4,3 +4,38 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin smartphone {
+  @media (min-width: 640px) {
+    @content;
+  }
+}
+
+@mixin smalldesktop {
+  @media (min-width: 1280px) {
+    @content;
+  }
+}
+
+@mixin desktop {
+  @media (min-width: 1440px) {
+    @content;
+  }
+}
+
+@mixin gridPage {
+  --columns: 2;
+  display: grid;
+  column-gap: 16px;
+  grid-template-columns: repeat(var(--columns), 1fr);
+
+  @include  smartphone {
+    --columns: 8;
+    column-gap: 24px;
+  }
+
+  // @include smalldesktop {
+  //   --columns: 12;
+  //   column-gap: 30px;
+  // }
```

Remove unused comments
```suggestion
  // @include smalldesktop {
  //   --columns: 12;
  //   column-gap: 30px;
  // }
```